test/everypolitician/popolo/event_test.rb

```diff
@@ -41,6 +41,7 @@ def test_accessing_legislative_periods
 
   def test_accessing_elections
     elections = popolo.elections
+    assert_instance_of Everypolitician::Popolo::Elections, elections
```

Comment:
I think we probably want to add a similar assertion to `test_accessing_legislative_periods` to ensure `popolo.legislative_periods` is returning the correct instance.